includes/tax-functions.php

```diff
@@ -101,10 +102,9 @@ function edd_get_tax_rate( $country = false, $state = false ) {
 		}
 	}
 
-	if( $rate > 1 ) {
-		// Convert to a number we can use
-		$rate = $rate / 100;
```

Comment:
This was originally done to enforce rates to be entered as a whole percentage and not `0.05` (to represent 5%).

If allowing `0.05`, we need to add a notice that shows a message to site owners when a rate is detected lower than that after installing this update so they can verify it's entered correctly, otherwise we could end up with a bunch of sites that are suddenly calculating very wrong tax amounts.